backend/api/search.py

```diff
@@ -77,7 +77,7 @@ def search_simple():
         raise ApiException(str(e), 400)
 
     if save:
-        return g.save_search(current_user, save, request_json, len(projects))
+        return g.save_search(current_user, json.loads(save), request_json, len(projects))
```

Comment:
this can still throw an exception and lead to a 500 when the string isn't valid json